source/release/4.2.0/index.html.md

```diff
@@ -0,0 +1,710 @@
+ - [BZ 1404509](https://bugzilla.redhat.com/1404509) <b>[RFE] Log collector should collect time diff for all hosts</b><br>When collecting sos reports from hosts, chrony and systemd sos plugins are now enabled collecting information about time synchronization. In addition, a new option --time-only has been added to ovirt-log-collector allowing to gather only information about time differences from the hosts without gathering full sos reports, saving a considerable amount of time for the operation.
+ - [BZ 1023455](https://bugzilla.redhat.com/1023455) <b>[CodeChange][RFE] log-collector should use the same code used by the engine for parsing configuration</b><br>
+
+#### oVirt Provider OVN
+
+ - [BZ 1472747](https://bugzilla.redhat.com/1472747) <b>Error while trying to authenticate ovirt-provider-ovn against Active Directory</b><br>Feature: <br>to authenticate ovirt-provider-ovn against an ldap server via user/password set<br><br>ovirt-admin-user-name=<admin_username><br>in /etc/ovirt-provider-ovn/conf.d<br><br>and use <admin_username>@<ad_domain>@<auth_profile> when defining the external provider in ovirt-engine.<br><br>Similarly, to authenticate with an active directory group set in /etc/ovirt-provider-ovn/conf.d :<br><br>[AUTH]<br>auth-plugin=auth.plugins.ovirt:AuthorizationByGroup<br><br>[OVIRT]<br>ovirt-admin-role-id=def00005-0000-0000-0000-def000000005<br>ovirt-admin-group-attribute-name=AAA_AUTHZ_GROUP_NAME;java.lang.String;0eebe54f-b429-44f3-aa80-4704cbb16835<br><br>and use <admin_username>@<ad_domain>@<auth_profile> when defining the external provider in ovirt-engine.<br><br>Reason: <br><br>Result:
+
+#### oVirt Engine Appliance
+
+ - [BZ 1463853](https://bugzilla.redhat.com/1463853) <b>[RFE] RHV-M appliance should meet NIST 800-53 partitioning requirements</b><br>Feature: <br>Modified disk partitioning scheme for RHVM-Appliance from 2 primary partitions ("/" and swap) to the one specified by NIST.<br><br>Reason: <br>Meet with NIST partitioning requirements<br><br>Result: <br>Updated disk partitions to be as follows:<br><br>/boot 1G (primary)<br>/home 1G (lvm)<br>/tmp 2G (lvm)<br>/var 20G (lvm)<br>/var/log 10G (lvm)<br>/var/log/audit 1G (lvm)<br>swap 8G (lvm)<br>/ 6G (primary)
+ - [BZ 1422982](https://bugzilla.redhat.com/1422982) <b>[RFE] add swap to rhevm-appliance</b><br>Feature: rhvm-appliance now includes a swap partition as part of the base install.<br><br>Reason: In some cases, the default memory allotment for rhvm-appliance may not be large enough to include support for user additions. Rather than increasing the memory for everyone, swap offers users more flexibility.<br><br>Result: There is a swap partition as part of the default build of rhvm-appliance
+
+### Known Issue
```

Comment:
In what sense is it "known issue"?

Reply:
In the sense that we know this affect the oVirt release and we haven't fixed it yet but manual action can be done to workaround. Bugs here may have been set with wrong value of doc type.